diff --git a/conf.d/agetty b/conf.d/agetty
index 03acee65..9f13ab8c 100644
--- a/conf.d/agetty
+++ b/conf.d/agetty
@@ -1,11 +1,11 @@
 # make agetty quiet
-#quiet="yes"
+quiet="yes"
 
 # Set the baud rate of the terminal line
 #baud=""
 
 # set the terminal type
-#term_type="linux"
+term_type="linux"
 
 # extra options to pass to agetty for this port
 #agetty_options=""
diff --git a/conf.d/bootmisc b/conf.d/bootmisc
index dd5b08e0..25ccb6d6 100644
--- a/conf.d/bootmisc
+++ b/conf.d/bootmisc
@@ -12,4 +12,4 @@ log_dmesg="YES"
 # Save the previous dmesg log to dmesg.old
 # This may be useful if you need to compare the current boot to the
 # previous one.
-#previous_dmesg=no
+previous_dmesg="YES"
diff --git a/etc/rc.conf b/etc/rc.conf
index ca0f9207..5499782f 100644
--- a/etc/rc.conf
+++ b/etc/rc.conf
@@ -13,13 +13,13 @@
 # boot so you can choose to start specific services. Set to "NO" to disable
 # this feature. This feature is automatically disabled if rc_parallel is
 # set to YES.
-#rc_interactive="YES"
+rc_interactive="NO"
 
 # If we need to drop to a shell, you can specify it here.
 # If not specified we use $SHELL, otherwise the one specified in /etc/passwd,
 # otherwise /bin/sh
 # Linux users could specify /sbin/sulogin
-#rc_shell=/bin/sh
+rc_shell=/usr/bin/sulogin
 
 # Do we allow any started service in the runlevel to satisfy the dependency
 # or do we want all of them regardless of state? For example, if net.eth0
@@ -48,15 +48,15 @@
 # /var/log/rc.log
 # NOTE: Linux systems require the devfs service to be started before
 # logging can take place and as such cannot log the sysinit runlevel.
-#rc_logger="NO"
+rc_logger="YES"
 
 # Through rc_log_path you can specify a custom log file.
 # The default value is: /var/log/rc.log
-#rc_log_path="/var/log/rc.log"
+rc_log_path="/var/log/rc.log"
 
 # If you want verbose output for OpenRC, set this to yes. If you want
 # verbose output for service foo only, set it to yes in /etc/conf.d/foo.
-#rc_verbose=no
+rc_verbose="NO"
 
 # By default we filter the environment for our running scripts. To allow other
 # variables through, add them here. Use a * to allow all variables through.
@@ -89,7 +89,7 @@
 # There variables are shared between many init scripts
 
 # Set unicode to NO to turn off unicode support for keyboards and screens.
-#unicode="YES"
+unicode="YES"
 
 # This is how long fuser should wait for a remote server to respond. The
 # default is 60 seconds, but  it can be adjusted here.
@@ -190,7 +190,7 @@
 # on Linux and Hurd, this is the number of ttys allocated for logins
 # It is used in the consolefont, keymaps, numlock and termencoding
 # service scripts.
-rc_tty_number=12
+rc_tty_number=3
 
 ##############################################################################
 # LINUX CGROUPS RESOURCE MANAGEMENT
diff --git a/init.d/bootmisc.in b/init.d/bootmisc.in
index 31f81a63..5ee33d93 100644
--- a/init.d/bootmisc.in
+++ b/init.d/bootmisc.in
@@ -230,7 +230,7 @@ start()
 						mv /var/log/dmesg /var/log/dmesg.old
 					fi
 					dmesg > /var/log/dmesg
-					chmod 640 /var/log/dmesg
+					chmod 644 /var/log/dmesg
 					;;
 			esac
 		fi
diff --git a/init.d/hostname.in b/init.d/hostname.in
index fc949336..02eff156 100644
--- a/init.d/hostname.in
+++ b/init.d/hostname.in
@@ -20,8 +20,8 @@ depend()
 start()
 {
 	local h source
-	if read -r h _ 2> /dev/null < @SYSCONFDIR@/hostname; then
-		source="@SYSCONFDIR@/hostname"
+	if read -r h _ 2> /dev/null < /etc/hostname; then
+		source="/etc/hostname"
 	elif [ -n "${hostname}" ]; then
 		h=${hostname}
 		source="@SYSCONFDIR@/conf.d/${RC_SVCNAME}"
diff --git a/meson.build b/meson.build
index 044e79db..fc9e0e01 100644
--- a/meson.build
+++ b/meson.build
@@ -85,7 +85,7 @@ bindir = get_option('prefix') / get_option('bindir')
 libdir = get_option('prefix') / get_option('libdir')
 pluginsdir = libdir / 'rc/plugins'
 libexecdir = get_option('prefix') / get_option('libexecdir')
-rc_libexecdir = libexecdir / 'rc'
+rc_libexecdir = libexecdir / 'openrc'
 rc_bindir = rc_libexecdir / 'bin'
 rc_sbindir = rc_libexecdir / 'sbin'
 rc_shdir = rc_libexecdir / 'sh'
diff --git a/meson_options.txt b/meson_options.txt
index 8cefbbbc..ce7d2410 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -29,5 +29,5 @@ option('sysvinit', type : 'boolean', value : false,
 option('zsh-completions', type : 'boolean',
   description : 'install zsh completions')
 option('agetty', type: 'array',
-  value: ['tty1', 'tty2', 'tty3', 'tty4', 'tty5', 'tty6'],
+  value: ['tty1', 'tty2', 'tty3'],
   description : 'agetty symlinks to install when sysvinit is also specified')
diff --git a/sh/init-early.sh.Linux.in b/sh/init-early.sh.Linux.in
index db0c1aa2..fbb34249 100644
--- a/sh/init-early.sh.Linux.in
+++ b/sh/init-early.sh.Linux.in
@@ -11,7 +11,7 @@
 
 : ${CONSOLE:=/dev/console}
 : ${RC_LIBEXECDIR:=@LIBEXECDIR@}
-: ${RC_CACHEDIR:=/var/cache/rc}
+: ${RC_CACHEDIR:=/var/cache/openrc}
 
 service_present()
 {
diff --git a/src/shared/misc.c b/src/shared/misc.c
index 3f0d3aae..d1e2c155 100644
--- a/src/shared/misc.c
+++ b/src/shared/misc.c
@@ -199,7 +199,7 @@ env_config(void)
 	}
 
 	if (!rc_is_user()) {
-		setenv("RC_CACHEDIR", "/var/cache/rc", 1);
+		setenv("RC_CACHEDIR", "/var/cache/openrc", 1);
 	} else {
 		const char *cache_home = getenv("XDG_CACHE_HOME");
 		char *cachedir = NULL;
diff --git a/support/sysvinit/inittab b/support/sysvinit/inittab
index 777df210..6d926423 100644
--- a/support/sysvinit/inittab
+++ b/support/sysvinit/inittab
@@ -31,9 +31,6 @@ su1:S:wait:/sbin/sulogin
 c1:12345:respawn:/sbin/agetty 38400 tty1 linux
 c2:2345:respawn:/sbin/agetty 38400 tty2 linux
 c3:2345:respawn:/sbin/agetty 38400 tty3 linux
-c4:2345:respawn:/sbin/agetty 38400 tty4 linux
-c5:2345:respawn:/sbin/agetty 38400 tty5 linux
-c6:2345:respawn:/sbin/agetty 38400 tty6 linux
 
 # SERIAL CONSOLES
 #s0:12345:respawn:/sbin/agetty -L 9600 ttyS0 vt100
diff --git a/tools/meson_runlevels.sh b/tools/meson_runlevels.sh
index 1c785618..9a059e1d 100755
--- a/tools/meson_runlevels.sh
+++ b/tools/meson_runlevels.sh
@@ -37,7 +37,7 @@ case "${os}" in
 		boot="${boot} ${boot_BSD} adjkerntz dumpon modules syscons"
 		;;
 	Linux)
-		boot="${boot} binfmt hwclock keymaps modules mtab procfs
+		boot="${boot} binfmt hwclock modules mtab procfs
 		save-keymaps save-termencoding seedrng termencoding"
 		;;
 	NetBSD)
@@ -45,7 +45,7 @@ case "${os}" in
 		;;
 esac
 
-default="local netmount"
+default="local"
 
 nonetwork="local"
 
