diff --git a/Makefile b/Makefile
index b19b01b..59af864 100644
--- a/Makefile
+++ b/Makefile
@@ -63,7 +63,7 @@ install_tmpfiles: install_common
 install_agetty: install_common
 	install $(FILEMODE) $(AGETTY_CONFD) $(DESTDIR)$(CONFDDIR)
 
-	for num in 1 2 3 4 5 6; do \
+	for num in 1 2 3; do \
 		$(LN) $(INITDDIR)/agetty $(DESTDIR)$(INITDDIR)/agetty.tty$$num; \
 		$(LN) $(INITDDIR)/agetty.tty$$num $(DESTDIR)$(RLVLDIR)/default/agetty.tty$$num; \
 	done
